cl/assets/static-global/css/override.css

```diff
@@ -1546,3 +1546,33 @@ textarea {
     float: right !important;
   }
 }
+
+/* Custom CSS for opinion coverage page */
+.timelines-dialog {
+    width: 95vw;
+}
+
+.legendG {
+    display: None;
+}
+
+#fullScreenModal {
+  width: 100vw;
+}
+
+#toc-container {
+  overflow-y: scroll;
+  height: 100vh;
+}
+
+#toc > ul > *:target {
+    background-color: white;
+}
+
+#toc > * > *:target {
+    background-color: white;
+}
```

Comment:
What are these about?

Reply:
the timeline chart has some odd decision it makes- we can override some of them here.

timelines-dialog - forces the width to match nearly the full width of the modal to give us an expansive view of the svg

.legendG removes the always required but not always useful legend - and not useful for us 

fullScreenModal - overrides bootstrap to give us the width we need to make the timeline chart work

toc container auto - (scroll is a mistake) auto and height are used to make the scrolling sidebar work 

the glowing yellow for target is not useful on the sidebar - it changes when we expand the table of contents - this is meant to ignore that weird css

Reply:
no - 

I double checked 

```
#toc > ul > *:target {
    background-color: white;
}
```

is required.

Reply:
I did remove the first one though

Reply:
HTML tweak to remove data target from sidebar was what was required.  The override css now has both rules removed.